curriculum/challenges/english/25-front-end-development/workshop-loan-qualification-checker/66c8ba975ee7230e29f6c4b7.md

```diff
@@ -0,0 +1,60 @@
+---
+id: 66c8ba975ee7230e29f6c4b7
+title: Step 16
+challengeType: 1
+dashedName: step-16
+---
+
+# --description--
+
+Use `let` to create a `qualificationMessage` variable and set it to `"You qualify for the following loan(s):\n \n' + loanMessage"`.
```

Comment:
We should make this more flexible because campers will know multiple ways to concatenate strings.
because technically this should be a valid answer

```js
`You qualify for the following loan(s):\n\n${loanMessage}`

```

we should update that message to allow for multiple correct answers